src/main/java/com/attacktimer/AttackTimerMetronomePlugin.java

```diff
@@ -171,7 +174,9 @@ private int getWeaponSpeed()
     {
         boolean isPlayerCasting = isPlayerCasting();
         int itemId = getWeaponId();
-        if (isPlayerCasting && itemId != 11907 && itemId != 12899) {
+        // Added id array of the following items: tridents, CG staves, and sanguinesti staff + kit.
+        Set<Integer> itemIds = new HashSet<>(Arrays.asList(11907, 12899, 23898, 23899, 23900, 25731, 22323, 23852, 23853, 23854));
+        if (isPlayerCasting && !itemIds.contains(itemId)) {
```

Comment:
Rather than checking against a set of itemIds that would need to be maintained as new staves are added that reuse a spellbook animation, could we instead check whether the equipped weapon is a powered staff?

Going off the configuration in WeaponType, type 28 should be used for powered staves (Accurate, Accurate, Longrange attack styles).

```suggestion
        private static final POWERED_STAFF_WEAPON_TYPE = 28;
        boolean isUsingPoweredStaff = client.getVarbitValue(Varbits.EQUIPPED_WEAPON_TYPE) == POWERED_STAFF_WEAPON_TYPE;
        if (isPlayerCasting && !isUsingPoweredStaff) {
```

Would be good to update the code comment below as well to reflect that its not just tridents - several powered staves share animations with wave-tier spells.